build-system/babel-config/helpers.js

```diff
@@ -115,8 +115,30 @@ function getReplaceGlobalsPlugin() {
   ];
 }
 
+/**
+ * @param {boolean} isProd
+ * @return {Array<string|Object>}
+ */
+function getPresetEnv(isProd) {
+  return [
+    '@babel/preset-env',
+    {
+      bugfixes: true,
+      modules: false,
+      loose: !isProd,
+      targets:
+        argv.esm || argv.sxg
+          ? {esmodules: true}
+          : isProd
+          ? {ie: 11, chrome: 41}
```

Comment:
nit: for prod can we get rid of ie11, and for dev `Last 2 versions` actually includes ie11 doesn't it?